commands/message.go

```diff
@@ -164,9 +163,9 @@ var msgWaitCmd = &cmds.Command{
 		found := false
 		err = GetPorcelainAPI(env).MessageWait(req.Context, msgCid, func(blk *types.Block, msg *types.SignedMessage, receipt *types.MessageReceipt) error {
 			found = true
-			sig, err2 := GetPorcelainAPI(env).ActorGetSignature(req.Context, msg.To, msg.Method)
-			if err2 != nil && err2 != mthdsig.ErrNoMethod && err2 != mthdsig.ErrNoActorImpl {
-				return errors.Wrap(err2, "Couldn't get signature for message")
+			sig, err := GetPorcelainAPI(env).ActorGetSignature(req.Context, msg.To, msg.Method)
```

Comment:
dont we need to not mask err here (hence err2 in original)?

Reply:
`err` is only masked in the context of the anonymous callback function where the original `err` is never used, so it's not an issue. If we need to access the original `err` again, we can change it back.